docs/btc-transactions-validity.md

```diff
@@ -1,23 +1,179 @@
 ## Introduction
 
-##  Prerequisites
+## Prerequisites
 
 ## Taproot outputs
 
-## Types of transactions
+## Building Transactions
 
-## Staking
+There are three special transaction types
 
-### Sending a TX
+- Staking Transaction
+- Unbonding Transaction
+- Slashing Transaction
 
-## Unbonding
+### Staking Transaction
 
-### Sending a TX
+A staking transaction is different to the other transactions. It creates a UTXO
+and can only be spent by specific paths. The paths are a predefined spending
+rules enforced by a one of the spending scripts [here] (#spending-scripts).
 
-## Spending taproot outputs
+Requirements:
 
-## Slashing
+1. **Inputs**: References previous UTXOs, provides BTC
+2. **Taproot Output (`staking_output`)**: The vault that locks BTC. Commits
+    to a script tree composed of the three scripts (timelock, unbondng and slashing.)
+    - **Taproot Public Key**: Created in the Taproot output, commits to `m`
+        (Merkle root)
+    - **Merkle Root**: Inside `Q`, commits to spending rules (timelock,
+        unbonding, slashing)
+    - **Internal Public Key**: Used to derive `Q`
+3. **`OP_RETURN` Output**:  Contains Babylon metadata (tag, `staker_pk`,
+    `finality_provider_pk`, `staking_time`, `version`, `global_parameters.tag`)
 
-### Sending a TX
+The following are **not yet** needed.
```

Comment:
I think I would delete this list. We do not know what inputs are used to create the staking transaction. It is possible the input to the staking transaction is previous staking output, then user would need to have those 😅